core/files/etc/nginx/includes/misp

```diff
@@ -19,12 +19,15 @@ fastcgi_hide_header X-Powered-By;
 
 location / {
     try_files $uri $uri/ /index.php$is_args$query_string;
+    fastcgi_read_timeout 3000;
```

Comment:
I don't like this duplication.

Why are these two lines needed in three different places?

Documentation here seems to specify that all these contexts are valid, hence we should add it to misp443 and misp80 (both because we might DISABLE_SSL).